### ISSUE TITLE
UX: cannot append to existing epubs

### ISSUE THREAD

**Issue by uqs** [plateaukao/einkbro#362]
Due to #357, I had to backup&restore the settings, which worked well. However, the next time to save an epub, the picker was listing all my existing epubs as zero size. So I deleted them from the picker and wanted to re-add them as existing epubs, but when you choose as "new" epub something that already exists, it will not append a chapter, but instead overwrite the epub.

So now I can no longer append some long-reads I'd like to save for later to existing epubs, I have to create new files. A workaround is probably making an epub copy, adding it to the picker, it gets overwritten, then I restore the backup.

It might be nicer to have einkbro always append to chosen epubs, and if the users wants to override them, they can first just delete the file via any file managing app, and then create it freshly.

wdyt?

**Comment by plateaukao** [comment]
@uqs 
yes, you can. Just use the picker, and choose first item to open the epub you saved before. 
einkbro will know it's already there, it will try to append the article. 

to be safer, you can back up your original epub, and give it a try.

**Comment by plateaukao** [comment]
as for the wrongly behavior after selecting epub file name with 0 size, I'll check why.
